documentation/en/source/pages/templates-spec/builders/builders-pxe.rst

```diff
@@ -0,0 +1,74 @@
+.. Copyright (c) 2007-2016 UShareSoft, All rights reserved
+
+.. _builder-pxe:
+
+PXE
+===
+
+Default builder type: ``PXE``
+
+Require Cloud Account: No
+
+This builder type is the default name provided by UForge AppCenter.
+
+.. note:: This builder type name can be changed by your UForge administrator. To get the available builder types, please refer to :ref:`command-line-format`
+
+The PXE builder provides information for building PXE images.
```

Comment:
You should explain in one more sentence how PXE image generated can be used (having a PXE boot, etc)